src/helpers/should-component-update.js

```diff
@@ -0,0 +1,8 @@
+function shouldComponentUpdate(prevProps, nextProps) {
```

Comment:
Also, breaking lifecycle methods out like this makes this harder to read imo. Should be easy to see in the main component file what makes the component update or not. Wdyt?

Reply:
@ellell okay, I understand that. I'll shift it around a bit.